database/mem_store.go

```diff
@@ -0,0 +1,34 @@
+package database
+
+import (
+	"github.com/google/uuid"
+	"sync"
+)
+
+type memStore struct {
+	requests      map[uuid.UUID]*RequestEntry
+	ethSendRawTxs map[uuid.UUID]*EthSendRawTxEntry
+	mutex         *sync.Mutex
+}
+
+func NewMemStore(requests map[uuid.UUID]*RequestEntry, ethSendRawTxs map[uuid.UUID]*EthSendRawTxEntry) Store {
```

Comment:
`NewMemStore` should not require arguments, but just create the empty maps inside

Reply:
updated